packages/kbn-optimizer/limits.yml

```diff
@@ -193,7 +193,7 @@ pageLoadAssetSize:
   visTypeTagcloud: 37575
   visTypeTimelion: 68883
   visTypeTimeseries: 55203
-  visTypeVega: 153573
+  visTypeVega: 571074
```

Comment:
This is a significant increase, is it intended?

Reply:
No this should be fixed in 62c3b1a6556cdf96a110a8d8a5e8db7fa684f280

Reply:
With this fix there's no changes to the `limits.yml` anymore.